### ISSUE TITLE
RuntimeError: There is no current event loop in thread 'MainThread'.

### ISSUE THREAD

**Issue by lambdaofgod** [eth-sri/lmql#6]
When I run `lmql run` on your joke example, or using `lml playground` I get that RuntimeError.

BTW thanks for a great initiative.

**Comment by lambdaofgod** [comment]
Error details when running `lml run` with a file:

```
Exception ignored in: <function AsyncOpenAIAPI.__del__ at 0x7faf6116aa70>
Traceback (most recent call last):
  File "/home/kuba/Projects/forks/lmql/src/lmql/runtime/bopenai/batched_openai.py", line 553, in __del__
  File "/etc/conda/envs/lmql/lib/python3.10/asyncio/events.py", line 656, in get_event_loop
RuntimeError: There is no current event loop in thread 'MainThread'.

```

**Comment by lambdaofgod** [comment]
Query finishes. In app the problem is that every next call does not work.

**Comment by lbeurerkellner** [comment]
I think I have observed this before on Linux systems specifically. This is very likely a asyncio clean-up issue. I will investigate this.

When you say next "next call", what exactly do you mean. Once the query finishes, `lmql run` should have terminated, right?

**Comment by lambdaofgod** [comment]
I mean in web app.

**Comment by lbeurerkellner** [comment]
Okay, thank. I will investigate further and report back here. If you run locally, the playground will actually re-run a new lmql process on each query. So it seems odd that it would block the next call.

**Comment by Paethon** [comment]
Seeing the exact same problem here as well (running on Arch Linux)

**Comment by lbeurerkellner** [comment]
I just released a bug fix release lmql==0.0.4.2, where this issue should be fixed. It would be really cool, if anyone who has experienced this bug, could briefly report back, if this is fixed.
